src/resources/MachineLearning/MachineLearningInterfaces.ts

```diff
@@ -27,6 +27,10 @@ export interface RegistrationModel extends GranularResource {
     exportOffset?: string;
     // The additional command line parameters that can be passed to the drill
     commandLineParameters?: string[];
+    // The additional configuration that can be passed to the model
+    extraConfig?: {
+        [key: string]: any;
+    };

```

Comment:
Maybe this? (tho i think both are fine really)

```suggestion
    extraConfig?: Record<string, any>
```

Reply:
Yes both are the same, `{ [key: string]: any}` is the idiomatic way of writing it. I can change it if you prefer, I don't mind at all

Reply:
As you prefer ;) I approved because both LGTM ;)